client/js/lounge.js

```diff
@@ -206,16 +206,18 @@ $(function() {
 			})
 		);
 		renderChannel(data.chan);
-		var chan = sidebar.find(".chan")
+
+		// Queries do not automatically focus, unless the user did a whois
```

Comment:
Wait, so does this mean if I click on a user and I already have a query open with them, it won't get focus? That seems confusing.